File: Dicelocked/Dicelocked/Commands/ICommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Dicelocked.Commands
{
    public enum Result
    {
        success, failure
    }
    public interface ICommand<T>
    {
        Result execution(T commandHandler);
    }
}

File: Server/Server/src/server/Commands/ICommand.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package server.Commands;

public interface ICommand<T> {

    public enum ResultCode {
        Success,
        Failure,
    }

    ResultCode execute(T commandHandler);
}
